### PR TITLE
Update get-clients.md

### PR DESCRIPTION
**Pull request by BillWeidenborner**
Removed section on Linux as it was deprecated in December 2022

### CODE REVIEW

File: Teams/get-clients.md [hunk 1]
```diff
@@ -102,77 +101,6 @@ If a user currently has a DMG installation of Teams and wants to replace it with
...
-```bash
-sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
-
-sudo sh -c 'echo -e "[teams]\nname=teams\nbaseurl=https://packages.microsoft.com/yumrepos/ms-teams\nenabled=1\nautorefresh=1\nkeeppackages=0\ntype=rpm-md\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/zypp/repos.d/teams.repo'
-
-sudo zypper refresh
-sudo zypper install teams
-```
-
 ---
```
Comment (JohanFreelancer9) [inline Teams/get-clients.md:105]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @BillWeidenborner, kindly commit the suggested change to remove the extra space. Many thanks.

**Review by BillWeidenborner — COMMENTED** [review]
Yes, looks good.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**